pipelines/build/common/openjdk_build_pipeline.groovy

```diff
@@ -540,6 +540,7 @@ class Build {
                                                                 context.MapParameter(name: 'PIPELINE_DISPLAY_NAME', value: "${displayName}"),
                                                                 context.MapParameter(name: 'APPLICATION_OPTIONS', value: "${appOptions}"),
                                                                 context.MapParameter(name: 'LABEL_ADDITION', value: additionalTestLabel),
+                                                                context.MapParameter(name: 'cause', value: "Remote triggered by job ${env.BUILD_URL}"),
```

Comment:
seemed odd this label is lower case?

Reply:
Ditto. Please convert "cause" to upper case.

Reply:
This label is used to append `&cause=Cause+Text` to be included in the recorded build cause for remote trigger job url, that is the variable name case is defined by trigger remote plugin. It is not part of our specific job parameter.

Reply:
I don't see any mention of "cause" here ? https://www.jenkins.io/doc/pipeline/steps/Parameterized-Remote-Trigger/

Reply:
It can be see on the remote triggered job configuration 
<img width="1480" alt="Screenshot 2023-03-03 at 11 32 47 AM" src="https://user-images.githubusercontent.com/18030625/222775461-5e5b8811-dbf4-4033-b2b3-2eafeb928147.png">

Reply:
I didn't know about that. Could you please run a build using your pipeline branch so we can see it in action?

Reply:
Yes, I did. https://ci.eclipse.org/temurin-compliance/job/Sophia_test/29/

Reply:
Interesting. Thanks for the link. I'm happy. LGTM.

Reply:
The idea is in the job https://ci.eclipse.org/temurin-compliance/job/AQA_Test_Pipeline/1207/ instead of see `Started by remote host 172.30.50.1 with note: null` you will see `Started by remote host 172.30.50.1 with note: Triggered by job https://ci.adoptium.net/job/build-scripts/job/release-openjdk**pipeline/**/`. I personally think that would be helpful especially for the triage.